processing/src/main/java/io/druid/query/aggregation/Histogram.java

```diff
@@ -58,6 +58,28 @@ public Histogram(float[] breaks, long[] bins, float min, float max)
     }
   }
 
+  public Histogram(Histogram other)
```

Comment:
Other complex aggregators rely solely on `copyFrom` is there an explicit need for having a constructor-copy method?